requirements/edx/github.in

```diff
@@ -74,3 +74,7 @@ git+https://github.com/edx/django-ratelimit-backend.git@v2.0.1a5#egg=django-rate
 
 git+https://github.com/open-craft/xblock-poll@da2d8fd21791a7af128595cf82bee83ee579e00f#egg=xblock-poll==1.9.6
 git+https://github.com/edx-solutions/xblock-drag-and-drop-v2@v2.2.10#egg=xblock-drag-and-drop-v2==2.2.10
+
+
+# Test new version of edx-search
+git+https://github.com/raccoongang/edx-search.git@golub-sergey/BD-19/transition-from-ES1.5-to-ES7
```

Comment:
This raises a very big question and concern for me. Explain why this is here, please.

Reply:
this requirement needs for testing `edx-search` with `edx-platform` changes. Sorry for disinformation.
I've made pr to the edx-platform without requirements update. https://github.com/edx/edx-platform/pull/24915
I suppose that edx-platfrom and edx-search changes are applied together

Reply:
So lead this process carefully, please.